AprilTagTrackers/Connection.cpp

```diff
@@ -24,7 +23,8 @@ void Connection::StartConnection()
         {
             return;
         }
-        Sleep(1000);
+        std::this_thread::sleep_for(std::chrono::milliseconds(1000));

```

Comment:
I think it would be nice to define a `Sleep` function when not on windows that encapsulates this.